rosdep/base.yaml

```diff
@@ -4372,6 +4372,16 @@ libmsgsl-dev:
   fedora: [guidelines-support-library-devel]
   gentoo: [dev-cpp/ms-gsl]
   ubuntu: [libmsgsl-dev]
+libmysql++-dev:
```

Comment:
Looks like the rosdep check action noticed `libmysql++-dev` is also available on fedora as the package `mysql++-devel`. Please add an entry for fedora to that key.

```yaml
  fedora: [mysql++-devel]
```

https://packages.fedoraproject.org/pkgs/mysql++/mysql++-devel/

Reply:
Ok, as the package is mysql++ the key should be named `mysql++devel` instead of libmysql++-dev.

Reply:
> Ok, as the package is mysql++ the key should be named mysql++-devel instead of libmysql++-dev.

No you had the key names correct. The key name should match the Ubuntu package name, just with the fedora entry added to it.

```
If a native package is available, the key name should match the name of the package in Ubuntu.
```

https://github.com/ros/rosdistro/blob/master/REVIEW_GUIDELINES.md

Reply:
Right, let me change it.